app/webpack/observations/show/ducks/controlled_terms.js

```diff
@@ -125,3 +157,12 @@ export function setControlledTermsForTaxon( taxon, terms = [] ) {
     dispatch( setControlledTerms( termsForTaxon( allTerms, taxon ) ) );
   };
 }
+
+export function fetchAnnotationsPanelPreferences( ) {
+  return ( dispatch, getState ) => {
+    const { config } = getState( );
+    const currentUser = config && config.currentUser;
+    const open = currentUser ? currentUser.prefers_hide_obs_show_annotations : false;
```

Comment:
minor thing here - prefers_hide_obs_show_annotations should be inverted here. The panel should be open if there's a user and that user does NOT prefer to hide the annotations panel (it's admittedly a bad preference name)

Reply:
Good catch, inverted this.